test/library/packages/csv/CSV.chpl

```diff
@@ -0,0 +1,167 @@
+module CSV {
+  record CSVIO {
```

Comment:
I know it's a prototype, but some simple doc strings on the module, record, and methods would go a long way helping some early users, e.g. myself :)